[2025-05-20T18:04:20.088+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-20T18:04:20.113+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_car_3.load_to_greenplum manual__2025-05-20T18:03:53.239669+00:00 [queued]>
[2025-05-20T18:04:20.130+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_car_3.load_to_greenplum manual__2025-05-20T18:03:53.239669+00:00 [queued]>
[2025-05-20T18:04:20.131+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-05-20T18:04:20.155+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_to_greenplum> on 2025-05-20 18:03:53.239669+00:00
[2025-05-20T18:04:20.166+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=207) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-20T18:04:20.169+0000] {standard_task_runner.py:72} INFO - Started process 209 to run task
[2025-05-20T18:04:20.171+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'test_car_3', 'load_to_greenplum', 'manual__2025-05-20T18:03:53.239669+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/test_car_3.py', '--cfg-path', '/tmp/tmph93xiq23']
[2025-05-20T18:04:20.174+0000] {standard_task_runner.py:105} INFO - Job 10: Subtask load_to_greenplum
[2025-05-20T18:04:20.240+0000] {task_command.py:467} INFO - Running <TaskInstance: test_car_3.load_to_greenplum manual__2025-05-20T18:03:53.239669+00:00 [running]> on host fe45ddcd054e
[2025-05-20T18:04:20.363+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_car_3' AIRFLOW_CTX_TASK_ID='load_to_greenplum' AIRFLOW_CTX_EXECUTION_DATE='2025-05-20T18:03:53.239669+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-20T18:03:53.239669+00:00'
[2025-05-20T18:04:20.365+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-20T18:04:20.387+0000] {test_car_3.py:142} INFO - >>> [START] load_to_greenplum
[2025-05-20T18:04:20.428+0000] {base.py:84} INFO - Retrieving connection 'postgres_default'
[2025-05-20T18:04:20.457+0000] {test_car_3.py:210} INFO - Successfully loaded 28/28 records to GreenPlum
[2025-05-20T18:04:20.458+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-20T18:04:20.478+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-20T18:04:20.479+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=test_car_3, task_id=load_to_greenplum, run_id=manual__2025-05-20T18:03:53.239669+00:00, execution_date=20250520T180353, start_date=20250520T180420, end_date=20250520T180420
[2025-05-20T18:04:20.547+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-20T18:04:20.593+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-20T18:04:20.597+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
